app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -235,11 +235,11 @@ def sshkeys
   end
 
   def flavors
-    collector.system_pools.each do |type|
+    collector.system_pools.each do |_key, value|
```

Comment:
If you don't need the key, it's slightly cheaper to do 

```suggestion
    collector.system_pools.each_value do |value|
```

Reply:
The key is throw-away so using `each_value` is perfect. Done.